src/main/scripts/src/app/shared/search-box/query-item/query-item-type.ts

```diff
@@ -17,11 +17,14 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import {until} from 'selenium-webdriver';
```

Comment:
Why this import?

Reply:
I don't know, why it is here. Maybe mistake..